website/common/script/libs/shops.js

```diff
@@ -187,16 +187,14 @@ shops.getMarketGearCategories = function getMarketGear (user, language) {
     text: i18n.t('none', language),
   };
 
-  let falseGear = filter(content.gear.flat, (gear) => {
-    let prevOwnedFalseGear = user.items.gear.owned[gear.key] === false && gear.klass !== user.stats.class;
-    let specialNonClassGear = !user.items.gear.owned[gear.key] &&
-                              content.classes.indexOf(gear.klass) < 0 &&
-                              content.classes.indexOf(gear.specialClass) < 0 &&
-                              (gear.canOwn && gear.canOwn(user));
-    return  prevOwnedFalseGear || specialNonClassGear;
+  let specialNonClassGear = filter(content.gear.flat, (gear) => {
+    return !user.items.gear.owned[gear.key] &&
+      content.classes.indexOf(gear.klass) < 0 &&
```

Comment:
what does indexOf < 0 means? if we're matching for not found, you should use === -1, same in the line below.

(This works but I think it's quite confusing, === -1 is what we use elesewhere)

Reply:
Ah, yeah. This was done by the previous author of these lines--I'll happily refactor that to standard.